superset/migrations/versions/be1e5bcb9f46_.py

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""empty message
+
+Revision ID: be1e5bcb9f46
+Revises: ('e557699a813e', '620241d1153f')
+Create Date: 2020-05-08 10:35:21.489571
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "be1e5bcb9f46"
+down_revision = ("e557699a813e", "620241d1153f")
+
+import sqlalchemy as sa
+from alembic import op
+
+
+def upgrade():
+    pass
+
+
+def downgrade():
+    pass
```

Comment:
I think this migration is redundant, and should be avoidable by pointing `e557699a813e` to `f9a30386bd74`

Reply:
Yeap, sure, that's a good point, will do